aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -42,9 +42,40 @@ class PGExtension:
     superuser: bool = True
 
 
+@dataclass
```

Comment:
Considering the use-case, I guess `frozen=True` is needed here for safety reasons, we don't want objects to be mutable.

Reply:
fixed